src/main/resources/templates/laDigitalAssister/householdPregnancyDueDate.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
+<head th:replace="fragments/head :: head(title=#{pregnantduedate.title})"></head>
+<body>
+<div class="page-wrapper">
+  <div th:replace="fragments/toolbar :: toolbar"></div>
+  <section class="slab">
+    <div class="grid">
+      <div th:replace="fragments/goBack :: goBackLink"></div>
+      <main id="content" role="main" class="form-card spacing-above-35">
+        <th:block th:replace="~{fragments/icons :: pregnant}"></th:block>
+        <th:block
+          th:replace="'fragments/cardHeader' :: cardHeader(header=#{pregnantduedate.header}, subtext=#{pregnantduedate.subheader})"/>
+        <th:block th:replace="'fragments/form' :: form(action=${formAction}, content=~{::formContent})">
+          <th:block th:ref="formContent">
+
+            <div class="form-card__content">
+              <div th:each="individual, iter: ${inputData.get('pregnanciesFullNames[]')}" class="spacing-below-15">
+                <hr>
+                <th:block th:with="pregnancyDueDateInputName=${'pregnancyDueDate' + T(formflow.library.inputs.FieldNameMarkers).DYNAMIC_FIELD_MARKER + individual.uuid}">
+                  <th:block th:replace="~{fragments/inputs/date ::
+                    date(inputName=${pregnancyDueDateInputName},
```

Comment:
Oh, I see... this is a little challenging.   Maybe we need to change the date fragment to work with a "suffix" to the name (where the "uuid" is the suffix passed in).  Hmmm...

Reply:
Maybe we could override the template and have the call be: 

```
${pregnancyDueDateInputName} would equal "pregnancyDueDate"
${dynamicFieldSuffix} would equal `T(formflow.library.inputs.FieldNameMarkers).DYNAMIC_FIELD_MARKER + individual.uuid`

date(inputName=${pregnancyDueDateInputName},
         inputNameSuffix = ${dynamicFIeldSuffix},   // optional field
         label = #{...},
         groupName='pregnancyDueDate'}
```

Reply:
@bseeger this is really helpful! Thank you